content/home/examples/an-application.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: An Application
+title: Sebuah Aplikasi
 order: 2
 domid: todos-example
 ---
 
-Using `props` and `state`, we can put together a small Todo application. This example uses `state` to track the current list of items as well as the text that the user has entered. Although event handlers appear to be rendered inline, they will be collected and implemented using event delegation.
+Menggunakan `props` dan `state`, kita dapat membuat sebuah aplikasi Todo simpel. Contoh ini menggunakan `state` untuk mencatat daftar *item* dan teks yang dimasukkan oleh pengguna. Walaupun *event handler* terlihat seperti di-*render* secara langsung, *event handler-event handler* tersebut akan dikumpulkan untuk kemudian nantinya diimplementasikan menggunakan *event delegation*.
```

Comment:
@regalius @arfianadam Ada saran untuk kata Todo ? sebaiknya kita _translasi secara literal_ atau kita biarkan saja memakai _loanwords_ ?

Reply:
I have no idea about what's the literal translation of `Todo` in this context that is easy to hear hahaha